### PR TITLE
Added new DB and fetched the data for db from api

### PR DESCRIPTION
**Pull request by dominikfranzen**

### CODE REVIEW

File: database_handling.py [hunk 1]
```diff
@@ -24,8 +21,32 @@
...
+    Column('quantity', Integer)
+)
 meta.create_all(engine)
 
-conn = engine.connect()
-ins = minifigure_parts.insert().values(key = 'alp0011346565', minifigure_id= 'alp001')
-result = conn.execute(ins)
\ No newline at end of file
+def insert_to_table(list_of_dicts, tablename):
+    if len(list_of_dicts) > 0:
```
Comment (fabianschmauder) [inline database_handling.py:35]: you can use an early return here to reduce nesting
```suggestion
    if len(list_of_dicts) == 0:
      return
```

File: database_handling.py [hunk 2]
```diff
@@ -24,8 +21,32 @@
...
+    with engine.connect() as connection:
+        query = connection.execute(minifigure_parts.select())
+        for row in query:
+            key = row[0]
+            minifigure_id = row[1]
+            part_id = row[2]
+            part_color_id = row[3]
+            print(part_id + ' in ' + part_color_id)
+
+#select_parts_to_check()
```
Comment (fabianschmauder) [inline database_handling.py:52]:
```suggestion
```

File: save_to_db.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import auth
+import json
+import database_handling as udb
+from datetime import datetime
+import db_secrets
+
+def save_to_db(category_number, identifier):
+    db_secrets.get_db_secrets()
+    if identifier == 'minifigure_parts' or identifier == 'minifigure_prices':
```
Comment (fabianschmauder) [inline save_to_db.py:9]: you can also use an early return here

File: save_to_db.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                'capture_date': capture_date,
+                'unit_price': entry['unit_price'],
+                'quantity': entry['quantity']
+            }
+            minifigure_prices_list.append(minifigure_price_item)
+            
+    return minifigure_prices_list
+
+#save_to_db(0, 'minifigure_parts')
+#save_to_db(0, 'minifigure_prices')
```
Comment (fabianschmauder) [inline save_to_db.py:67]:
```suggestion
```

**Review by fabianschmauder — COMMENTED** [review]
just small changes

File: save_to_db.py (hunk 3 above)
Comment (dominikfranzen) [reply save_to_db.py:9]: I will have to split this to separate files due to integration into lambda functions, i guess

**Pull request merged**